ocrd_monitor/README.md

```diff
@@ -0,0 +1,60 @@
+# OCRD-Monitor
+
+The OCRD-Monitor application is used to monitor the progress and results of OCR-D jobs.
+In order to run the application you need to have an OCR-D Controller as well as the Dozzle container for viewing logs set up.
+To launch the monitor, first create a script containing the following code:
+
+```bash
+export OCRD_BROWSER__MODE=<native or docker>
+export OCRD_BROWSER__WORKSPACE_DIR=<dir-containing-ocrd-workspaces>
+export OCRD_BROWSER__PORT_RANGE="[9000,9100]"
+export OCRD_CONTROLLER__JOB_DIR=<dir-containing-ocrd-job-files>
+export OCRD_CONTROLLER__HOST=<url-to-ocrd-controller>
+export OCRD_CONTROLLER__PORT=<port-of-ocrd-controller>
+export OCRD_CONTROLLER__USER=<name-of-user-on-ocrd-controller>
+export OCRD_CONTROLLER__KEYFILE=<path-to-private-ssh-key>
+export OCRD_LOGVIEW__PORT=<port-of-dozzle>
+
+cd ocrd_monitor
+uvicorn "ocrdmonitor.main:app" --reload
+```
```

Comment:
We already have that in init.sh – I don't think this Readme should document the web application independently, for it *is* already containerised and only makes sense as part of the overall ocrd_kitodo setup.

```suggestion
In order to run the application you need to have set up an OCR-D Controller service (for runtime statistics)
as well as a local Dozzle container (for viewing logs).

The Monitor is run via `docker compose`. Environment variables to be defined externally (via shell or .env, see [ocrd_kitodo configuration](https://slub.github.io/ocrd_kitodo/setup/configure-modules/)) include:
- `MONITOR_IMAGE`: name and tag of the Docker image to be built/pulled
- `MONITOR_HOST`: name of the Monitor container on the (Docker) network
- `MONITOR_PORT_WEB`: TCP port to expose the web server at
- `MONITOR_PORT_LOG`: TCP port to expose the log viewer at
- `MONITOR_DATA`: host path of data directory to mount
- `MANAGER_KEY`: host path of SSH key to use for login to the Controller
- `CONTROLLER_HOST`: name/address of the Controller service on the network
- `CONTROLLER_PORT_SSH`: TCP port of the Controller service on the network

```

Reply:
I actually think that the monitor is rather independent and most of the dependencies to the whole ocrd-kitodo setup are due to the things I carried over from the previous flask implementation.

There is no actual dependency on the OCR-D Manager. The only things we need are stored on the filesystem. 
And the only reason we need the OCR-D Controller is because of the job status checks, which could also be handled by the manager since it already handles collecting information about job completion in `/run/lock/ocrd.jobs`.
In fact, I think this is the way it should be handled since we are currently collecting job information from two different sources. We're reading general job information from the files in `/run/lock/ocrd.jobs` and then logging into the controller to collect process information. I believe it would make our whole setup easier to handle if that information came from the same source.

So if we collect all job information from the filesystem (or some database later on), then suddenly the monitor neither needs to know about the manager nor the controller.